### ISSUE TITLE
filament advanced pause not working and crashes after approx 10 mins - due to screen sleeping

### ISSUE THREAD

**Issue by captaincreative** [klack/marlin#164]
### Bug Description

filament advanced pause not working and crashes after approx 10 mins - due to screen sleeping

### Bug Timeline

new bug 

### Expected behavior

pause for as long as needed to change filament, not really worth having filament pause if you need to watch it , idea to leave over night and swap filament in morning if required.

### Actual behavior

pause works but only for 10-15 mins approx..

### Steps to Reproduce

1) cut filament so it runs out
2) wait to see when screen goes off
3) awake screen and filament pause has crashed

### Version of Marlin Firmware

2.0.9.2

### Printer model

hictop d3

### Bed Leveling

UBL Bilinear mesh

### Your Slicer

Cura

### Host Software

None

**Comment by klack** [comment]
Removal of screen sleeping cause a bug where temp cannot be changed during sdprint, need to fix.

**Comment by captaincreative** [comment]
yes i have noticed this aswell, i can not edit the temps during sd card print

**Comment by klack** [comment]
Yes will take a look at this in the evening.

**Comment by klack** [comment]
`tjc x3 202301052300` posted in gdrive for testing
